### PR TITLE
update splash on real device

### PR DESCRIPTION
**Pull request by rynop**
per https://github.com/NativeScript/nativescript-cli/issues/3159

### CODE REVIEW

File: docs/tooling/publishing/creating-launch-screens-ios.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ As described in [iOS Human Interface Guidelines](https://developer.apple.com/lib
 In NativeScript, your application comes with predefined template settings and images for these steps.
 In this article, we are going to introduce the workflow to create your own launch screens.
 
-> **WARNING**: Occasionally, the iOS operating system caches your application’s icons and launch screens. If you’re updating icons or launch screens and not seeing your changes, delete the application from your device or emulator and redeploy.
+> **WARNING**: Occasionally, the iOS operating system caches your application’s icons and launch screens. If you’re updating icons or launch screens and not seeing your changes, delete the application from your device or emulator and redeploy.  If on real device: remove `platforms` folder from your project, delete app, restart device, redelploy.
```
Comment (etabakov) [inline docs/tooling/publishing/creating-launch-screens-ios.md]:
```suggestion
> **WARNING**: Occasionally, the iOS operating system caches your application’s icons and launch screens. If you’re updating icons or launch screens and not seeing your changes, delete the application from your device or emulator and redeploy. If on a real device: remove `platforms` folder from your project, delete app, restart device, redeploy.
```

Comment (etabakov) [reply docs/tooling/publishing/creating-launch-screens-ios.md]: There is a redundant whitespace and a typo.

**Review by etabakov — CHANGES_REQUESTED** [review]
Thanks for the contribution. There are two minor suggestions from my side.

File: docs/tooling/publishing/creating-launch-screens-ios.md (hunk 1 above)
Comment (rynop) [reply docs/tooling/publishing/creating-launch-screens-ios.md]: oops - yea good catch!

Comment (etabakov) [reply docs/tooling/publishing/creating-launch-screens-ios.md]: Do you mind to apply the changes suggested, so we can move this forward? Thanks!

**Comment by rynop** [comment]
I committed your suggestion (d05aad2) but it still says changes requested? Any ideas?

**Review by etabakov — APPROVED** [review]

**Comment by etabakov** [comment]
Thanks for the effort! We have a special program for our first-time contributors and you are eligible to participate in it. If you haven't applied already, all you need to do to claim your prize is to fill our [Application Form](https://goo.gl/forms/lPKpXzG4F1EhfKM72).

Regards from the whole team 🥇

**Comment by rynop** [comment]
Hi @etabakov - filled out the app form you mentioned back in Jan, haven't gotten anything.  I love NS and would like to represent/market it....

**Comment by etabakov** [comment]
Hey @rynop - sorry about that! The package that we sent to you in January was returned to us a couple of weeks ago by the postal service as undeliverable. We will resend it with the next batch in the next couple of days.

**Comment by rynop** [comment]
@etabakov my bad - I must have mistyped or something.  I sent you a PM on twitter to `@emiltabakov` with my address.

**Pull request merged**